src/main/java/Bot_Memory.java

```diff
@@ -0,0 +1,132 @@
+/*
+*  "Memory/Storage"
+*
+* */
+
+
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Bot_Memory {
+    private static File file;
+
+    public static void memory_Init(){
```

Comment:
Methods should be verbs written in camelCase, e.g. maybe initializeMemory() ?

Reply:
I changed it but it was not pushed yet